platform-ui-skin/src/main/webapp/skin/less/platform/skin/enterprise.less

```diff
@@ -927,6 +927,7 @@ input[disabled],
   box-shadow: none;
   max-height: 115px;
   text-overflow: ellipsis;
+  width: 95%;
```

Comment:
are you sure this modification is still necessary ? It seems taht you set width in ecms PR https://github.com/exoplatform/ecms/pull/1616/files

Reply:
after many tests and changes, I think it is not necessary this modification because like what you said it was already modified in ecms PR @rdenarie

Reply:
So please close this PR.
Thanks